### PR TITLE
fix ecs ipv4

### PR DESCRIPTION
**Pull request by leoparente**

### CODE REVIEW

**Review by rboucher-me — APPROVED** [review]

File: src/handlers/dns/DnsStreamHandler.h [hunk 1]
```diff
@@ -130,7 +130,7 @@ class DnsMetricsBucket final : public visor::AbstractMetricsBucket
         , _dns_topQname3("dns", "qname", {"top_qname3"}, "Top QNAMES, aggregated at a depth of three labels")
         , _dns_topNX("dns", "qname", {"top_nxdomain"}, "Top QNAMES with result code NXDOMAIN")
         , _dns_topREFUSED("dns", "qname", {"top_refused"}, "Top QNAMES with result code REFUSED")
-        , _dns_topSizedQnameResp("dns", "qname", {"top_qname_by_resp_size"}, "Top QNAMES by response size")
+        , _dns_topSizedQnameResp("dns", "qname", {"top_qname_by_resp_bytes"}, "Top QNAMES by response size in bytes")
```
Comment (rboucher-me) [inline src/handlers/dns/DnsStreamHandler.h]: Suggest we edit the description to "Top QNAMES by response volume in bytes"

File: src/handlers/net/NetStreamHandler.h [hunk 2]
```diff
@@ -113,8 +113,8 @@ class NetworkMetricsBucket final : public visor::AbstractMetricsBucket
         , _payload_size("packets", {"payload_size"}, "Quantiles of payload sizes, in bytes")
         , _rate_in("packets", {"rates", "pps_in"}, "Rate of ingress in packets per second")
         , _rate_out("packets", {"rates", "pps_out"}, "Rate of egress in packets per second")
-        , _throughput_in("payload", {"rates", "bps_in"}, "Rate of ingress packets size in bytes per second")
-        , _throughput_out("payload", {"rates", "bps_out"}, "Rate of egress packets size in bytes per second")
+        , _throughput_in("payload", {"rates", "bytes_in"}, "Rate of ingress packets size in bytes per second")
```
Comment (rboucher-me) [inline src/handlers/net/NetStreamHandler.h]: suggest we change to: "Rate of ingress packets by size in bytes per second"

File: src/handlers/net/NetStreamHandler.h [hunk 3]
```diff
@@ -113,8 +113,8 @@ class NetworkMetricsBucket final : public visor::AbstractMetricsBucket
...
+        , _throughput_out("payload", {"rates", "bytes_out"}, "Rate of egress packets size in bytes per second")
```
Comment (rboucher-me) [inline src/handlers/net/NetStreamHandler.h]: suggest we change to: "Rate of egress packets by size in bytes per second"

File: src/handlers/flow/FlowStreamHandler.h [hunk 4]
```diff
@@ -151,8 +151,8 @@ class FlowMetricsBucket final : public visor::AbstractMetricsBucket
         , _topByBytes("bytes")
         , _topByPackets("packets")
         , _payload_size(FLOW_SCHEMA, {"payload_size"}, "Quantiles of payload sizes, in bytes")
-        , _rate(FLOW_SCHEMA, {"rates", "pps"}, "Rate of combined flow packets per second")
-        , _throughput("payload", {"rates", "bps"}, "Rate of combined flow bytes per second")
+        , _rate(FLOW_SCHEMA, {"rates", "packets"}, "Rate of combined flow packets per second")
```
Comment (rboucher-me) [inline src/handlers/flow/FlowStreamHandler.h]: We should probably discuss this. My suggestion would be "Rate of total flows in packets per second"

File: src/handlers/flow/FlowStreamHandler.h [hunk 5]
```diff
@@ -151,8 +151,8 @@ class FlowMetricsBucket final : public visor::AbstractMetricsBucket
...
+        , _throughput("payload", {"rates", "bytes"}, "Rate of combined flow bytes per second")
```
Comment (rboucher-me) [inline src/handlers/flow/FlowStreamHandler.h]: We should probably discuss this. My suggestion would be "Rate of total flows in bytes per second"

**Review by rboucher-me — CHANGES_REQUESTED** [review]

File: src/handlers/flow/FlowStreamHandler.h (hunk 4 above)
Comment (weyrick) [reply src/handlers/flow/FlowStreamHandler.h]: hm for me that's still ambiguous. how about "Rate of combined flow throughput in packets per second"

File: src/handlers/flow/FlowStreamHandler.h (hunk 5 above)
Comment (weyrick) [reply src/handlers/flow/FlowStreamHandler.h]: similar "Rate of combined flow throughput in bytes per second"

File: src/handlers/net/NetStreamHandler.h (hunk 2 above)
Comment (weyrick) [reply src/handlers/net/NetStreamHandler.h]: why "by size"? isn't this also just throughupt?

"Rate of ingress throughput in bytes per second"

File: src/handlers/net/NetStreamHandler.h (hunk 3 above)
Comment (weyrick) [reply src/handlers/net/NetStreamHandler.h]: "Rate of egress throughput in bytes per second"

**Review by weyrick — CHANGES_REQUESTED** [review]

File: src/handlers/dns/DnsStreamHandler.h (hunk 1 above)
Comment (leoparente) [reply src/handlers/dns/DnsStreamHandler.h]: done

File: src/handlers/net/NetStreamHandler.h (hunk 2 above)
Comment (leoparente) [reply src/handlers/net/NetStreamHandler.h]: did @weyrick changes. @rboucher-ns1  thoughts?

File: src/handlers/net/NetStreamHandler.h (hunk 3 above)
Comment (leoparente) [reply src/handlers/net/NetStreamHandler.h]: did @weyrick changes. @rboucher-ns1  thoughts?

File: src/handlers/flow/FlowStreamHandler.h (hunk 5 above)
Comment (leoparente) [reply src/handlers/flow/FlowStreamHandler.h]: did @weyrick changes. @rboucher-ns1  thoughts?

File: src/handlers/net/NetStreamHandler.h (hunk 2 above)
Comment (rboucher-me) [reply src/handlers/net/NetStreamHandler.h]: I'm ok with this. I was confused about what the quantiles were actually about...

File: src/handlers/net/NetStreamHandler.h (hunk 3 above)
Comment (rboucher-me) [reply src/handlers/net/NetStreamHandler.h]: I'm ok with this

File: src/handlers/flow/FlowStreamHandler.h (hunk 5 above)
Comment (rboucher-me) [reply src/handlers/flow/FlowStreamHandler.h]: Ok for now. We'll probably revisit this with the upcoming flow work.

**Review by rboucher-me — APPROVED** [review]

File: src/handlers/flow/FlowStreamHandler.h (hunk 5 above)
Comment (leoparente) [reply src/handlers/flow/FlowStreamHandler.h]: cool

**Review by weyrick — APPROVED** [review]

**Pull request merged**